contracts/common/DomainRoles.sol

```diff
@@ -15,27 +15,27 @@
   along with The Colony Network. If not, see <http://www.gnu.org/licenses/>.
 */
 
-pragma solidity 0.5.8; // ignore-swc-103
+pragma solidity 0.7.3; // ignore-swc-103
 
 import "./../../lib/dappsys/roles.sol";
 
 
 contract DomainRoles is DSRoles {
-  mapping(address=>mapping(uint256=>bytes32)) internal _user_domain_roles;
+  mapping(address=>mapping(uint256=>bytes32)) internal userDomainRoles;
```

Comment:
I followed the snake case convention here because that's what `DSRoles` used and this is basically an override of that, but I could also see how this crosses the threshold into the "Colony" contracts and so we should adopt the Colony style.